t/TestWasm.pm

```diff
@@ -263,6 +265,12 @@ add_block_preprocessor(sub {
                 $timeout += 30;
                 $timeout = max($timeout, 30);
             }
+
+        } else {
+            if ($osname =~ m/darwin/ && $archname =~ m/arm64/) {
```

Comment:
This could be an "else if" instead of else, if?

Reply:
Indeed.